examples/deploy/terraform/cluster/main.tf

```diff
@@ -55,6 +56,9 @@ module "irsa_external_dns" {
   use_cluster_odc_idp = local.is_eks_account_same
   eks_info            = module.eks.info
   external_dns        = var.irsa_external_dns
+  region              = local.region
```

Comment:
Why does this `irsa` module instance which is used to enable irsa_external_dns need these now.